CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -29,6 +33,15 @@ hopperbinGUI.Adornee = hopperbinPart
 hopperbinGUI.ToolPunchThroughDistance = 1000
 hopperbinGUI.Name = "HopperBinGUI"
 
+local healthbarBack = Instance.new("Frame", hopperbinGUI)
```

Comment:
I believe so, but I'll test it to make sure. The only check is for VR when requiring this, and all signs point to the code running no matter what's enabled.